packages/validate/test-suites/ValidateMixin.suite.js

```diff
@@ -812,6 +812,28 @@ export function runValidateMixinSuite(customConfig) {
         expect(el.validationStates.error).to.eql({});
       });
 
+      it('clears current validation results when validators array updated', async () => {
+        const validators = [new Required()];
+        const el = await fixture(html`
+          <${tag}
+            .validators=${validators}
+          >${lightDom}</${tag}>
+        `);
+        el.modelValue = undefined;
```

Comment:
Thanks for the fix :)
Since we should test whether a new validation is triggered when we change the `.validators`,
it would be better to leave out all the lines with `el.modelValue = undefined;`?
(Especially since a modelValue change also creates a recomputation of the validity state.)